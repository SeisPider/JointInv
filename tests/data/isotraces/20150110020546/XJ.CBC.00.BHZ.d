SURF96 R U X -1 75.00000 3.70435 0.18368
SURF96 R U X -1 70.00000 3.73951 0.17471
SURF96 R U X -1 65.00000 3.87810 0.17447
SURF96 R U X -1 60.00000 3.82665 0.15681
SURF96 R U X -1 55.00000 3.73516 0.13695
SURF96 R U X -1 50.00000 3.68778 0.12136
SURF96 R U X -1 48.00000 3.68148 0.11611
SURF96 R U X -1 46.00000 3.66675 0.11038
SURF96 R U X -1 44.00000 3.64455 0.10431
SURF96 R U X -1 42.00000 3.62211 0.09834
SURF96 R U X -1 40.00000 3.60290 0.09267
SURF96 R U X -1 40.00000 3.15881 0.07123
SURF96 R U X -1 38.00000 3.56544 0.08622
SURF96 R U X -1 36.00000 3.48479 0.07803
SURF96 R U X -1 34.00000 3.40400 0.07031
SURF96 R U X -1 32.00000 3.38600 0.06548
SURF96 R U X -1 29.00000 3.43376 0.06103
SURF96 R U X -1 28.00000 3.42925 0.05877
SURF96 R U X -1 28.00000 3.08703 0.04762
SURF96 R U X -1 27.00000 3.40643 0.05592
SURF96 R U X -1 27.00000 3.07757 0.04564
SURF96 R U X -1 26.00000 3.37700 0.05292
SURF96 R U X -1 26.00000 3.16815 0.04658
