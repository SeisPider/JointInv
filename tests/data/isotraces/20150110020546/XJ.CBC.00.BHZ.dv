MFT96 R U T -1 75.852    3.7953    1.0000    5603.0     0.1447E-05   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 74.473    3.7981    1.0000    5603.0     0.1569E-05   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 73.143    3.8003    1.0000    5603.0     0.1696E-05   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 71.860    3.8021    1.0000    5603.0     0.1827E-05   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 70.621    3.8034    1.0000    5603.0     0.1961E-05   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 69.424    3.8042    1.0000    5603.0     0.2099E-05   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 68.267    3.8046    1.0000    5603.0     0.2240E-05   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 67.148    3.8046    1.0000    5603.0     0.2385E-05   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 66.065    3.8041    1.0000    5603.0     0.2532E-05   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 65.016    3.8031    1.0000    5603.0     0.2681E-05   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 64.000    3.8018    1.0000    5603.0     0.2833E-05   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 63.015    3.8001    1.0000    5603.0     0.2988E-05   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 62.061    3.7980    1.0000    5603.0     0.3144E-05   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 61.134    3.7955    1.0000    5603.0     0.3302E-05   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 60.235    3.7926    1.0000    5603.0     0.3462E-05   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 59.362    3.7894    1.0000    5603.0     0.3623E-05   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 58.514    3.7859    1.0000    5603.0     0.3786E-05   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 57.690    3.7820    1.0000    5603.0     0.3950E-05   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 56.889    3.7778    1.0000    5603.0     0.4116E-05   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 56.110    3.7734    1.0000    5603.0     0.4283E-05   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 55.351    3.7686    1.0000    5603.0     0.4452E-05   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 54.613    3.7636    1.0000    5603.0     0.4624E-05   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 53.895    3.7583    1.0000    5603.0     0.4799E-05   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 53.195    3.7528    1.0000    5603.0     0.4977E-05   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 52.513    3.7470    1.0000    5603.0     0.5160E-05   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 51.848    3.7411    1.0000    5603.0     0.5349E-05   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 51.200    3.7349    1.0000    5603.0     0.5545E-05   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 50.568    3.7285    1.0000    5603.0     0.5748E-05   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 49.951    3.7219    1.0000    5603.0     0.5959E-05   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 49.349    3.7152    1.0000    5603.0     0.6179E-05   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 48.762    3.7083    1.0000    5603.0     0.6407E-05   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 48.188    3.7013    1.0000    5603.0     0.6645E-05   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 47.628    3.6941    1.0000    5603.0     0.6892E-05   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 47.080    3.6868    1.0000    5603.0     0.7146E-05   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 46.545    3.6794    1.0000    5603.0     0.7409E-05   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 46.022    3.6720    1.0000    5603.0     0.7678E-05   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 45.511    3.6644    1.0000    5603.0     0.7954E-05   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 45.011    3.6567    1.0000    5603.0     0.8236E-05   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 44.522    3.6490    1.0000    5603.0     0.8524E-05   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 44.043    3.6413    1.0000    5603.0     0.8818E-05   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 43.574    3.6335    1.0000    5603.0     0.9119E-05   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 43.116    3.6256    1.0000    5603.0     0.9427E-05   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 42.667    3.6178    1.0000    5603.0     0.9744E-05   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 42.227    3.6100    1.0000    5603.0     0.1007E-04   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 41.796    3.6021    1.0000    5603.0     0.1041E-04   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 41.374    3.5943    1.0000    5603.0     0.1076E-04   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 40.960    3.5864    1.0000    5603.0     0.1112E-04   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 40.554    3.5787    1.0000    5603.0     0.1149E-04   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 40.157    3.5709    1.0000    5603.0     0.1187E-04   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 39.767    3.5632    1.0000    5603.0     0.1226E-04   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 39.385    3.5556    1.0000    5603.0     0.1266E-04   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 39.010    3.5480    1.0000    5603.0     0.1306E-04   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 38.642    3.5406    1.0000    5603.0     0.1345E-04   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 38.280    3.5332    1.0000    5603.0     0.1383E-04   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 37.926    3.5258    1.0000    5603.0     0.1419E-04   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 37.578    3.5186    1.0000    5603.0     0.1454E-04   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 37.236    3.5116    1.0000    5603.0     0.1486E-04   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 36.901    3.5046    1.0000    5603.0     0.1515E-04   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 36.571    3.4978    1.0000    5603.0     0.1541E-04   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 36.248    3.4911    1.0000    5603.0     0.1564E-04   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 35.930    3.4845    1.0000    5603.0     0.1583E-04   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 35.617    3.4781    1.0000    5603.0     0.1598E-04   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 35.310    3.4718    1.0000    5603.0     0.1610E-04   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 35.009    3.4658    1.0000    5603.0     0.1619E-04   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 34.712    3.4599    1.0000    5603.0     0.1625E-04   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 34.420    3.4541    1.0000    5603.0     0.1629E-04   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 34.133    3.4486    1.0000    5603.0     0.1632E-04   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 33.851    3.4433    1.0000    5603.0     0.1633E-04   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 33.574    3.4381    1.0000    5603.0     0.1633E-04   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 33.301    3.4332    1.0000    5603.0     0.1632E-04   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 33.032    3.4285    1.0000    5603.0     0.1630E-04   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 32.768    3.4240    1.0000    5603.0     0.1628E-04   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 32.508    3.4198    1.0000    5603.0     0.1626E-04   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 32.252    3.4158    1.0000    5603.0     0.1622E-04   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 32.000    3.4120    1.0000    5603.0     0.1618E-04   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 31.752    3.4085    1.0000    5603.0     0.1611E-04   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 31.508    3.4053    1.0000    5603.0     0.1602E-04   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 31.267    3.4023    1.0000    5603.0     0.1591E-04   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 31.030    3.3996    1.0000    5603.0     0.1578E-04   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 30.797    3.3972    1.0000    5603.0     0.1561E-04   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 30.567    3.3951    1.0000    5603.0     0.1541E-04   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 30.341    3.3933    1.0000    5603.0     0.1518E-04   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 30.118    3.3917    1.0000    5603.0     0.1492E-04   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 29.898    3.3906    1.0000    5603.0     0.1464E-04   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 29.681    3.3897    1.0000    5603.0     0.1434E-04   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 29.468    3.3891    1.0000    5603.0     0.1403E-04   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 29.257    3.3889    1.0000    5603.0     0.1371E-04   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 29.050    3.3891    1.0000    5603.0     0.1339E-04   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 28.845    3.3896    1.0000    5603.0     0.1307E-04   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 28.643    3.3905    1.0000    5603.0     0.1276E-04   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 28.444    3.3917    1.0000    5603.0     0.1245E-04   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 28.248    3.3933    1.0000    5603.0     0.1216E-04   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 28.055    3.3954    1.0000    5603.0     0.1187E-04   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 27.864    3.3978    1.0000    5603.0     0.1160E-04   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 27.676    3.4007    1.0000    5603.0     0.1132E-04   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 27.490    3.4039    1.0000    5603.0     0.1104E-04   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 27.307    3.4077    1.0000    5603.0     0.1076E-04   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 27.126    3.4118    1.0000    5603.0     0.1046E-04   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 26.947    3.4165    1.0000    5603.0     0.1014E-04   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 26.771    3.4216    1.0000    5603.0     0.9796E-05   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 26.597    3.4271    1.0000    5603.0     0.9429E-05   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 26.426    3.4332    1.0000    5603.0     0.9036E-05   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 26.256    3.4399    1.0000    5603.0     0.8616E-05   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 26.089    3.4470    1.0000    5603.0     0.8170E-05   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 75.000    3.7970    1.0000    5603.0     0.1522E-05   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 70.000    3.8038    1.0000    5603.0     0.2033E-05   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 65.000    3.8031    1.0000    5603.0     0.2684E-05   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 60.000    3.7917    1.0000    5603.0     0.3506E-05   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 55.000    3.7662    1.0000    5603.0     0.4534E-05   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 50.000    3.7224    1.0000    5603.0     0.5942E-05   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 48.000    3.6989    1.0000    5603.0     0.6728E-05   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 46.000    3.6716    1.0000    5603.0     0.7690E-05   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 44.000    3.6406    1.0000    5603.0     0.8846E-05   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 42.000    3.6058    1.0000    5603.0     0.1025E-04   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 40.000    3.5678    1.0000    5603.0     0.1203E-04   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 38.000    3.5274    1.0000    5603.0     0.1412E-04   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 36.000    3.4859    1.0000    5603.0     0.1578E-04   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 34.000    3.4461    1.0000    5603.0     0.1632E-04   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 32.000    3.4120    1.0000    5603.0     0.1618E-04   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 30.000    3.3911    1.0000    5603.0     0.1477E-04   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 29.000    3.3892    1.0000    5603.0     0.1331E-04   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 28.000    3.3961    1.0000    5603.0     0.1179E-04   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 27.000    3.4151    1.0000    5603.0     0.1023E-04   -5.655000   68.361298   43.521099   81.439201 0  1 COMMENT
