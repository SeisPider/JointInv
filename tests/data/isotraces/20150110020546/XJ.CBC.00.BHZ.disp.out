SURF96 R U T -1 75.000    3.7970    1.0000    
SURF96 R C T -1 75.000    6.2162    1.0000    
SURF96 R U T -1 70.000    3.8038    1.0000    
SURF96 R C T -1 70.000    5.9636    1.0000    
SURF96 R U T -1 65.000    3.8031    1.0000    
SURF96 R C T -1 65.000    5.7311    1.0000    
SURF96 R U T -1 60.000    3.7917    1.0000    
SURF96 R C T -1 60.000    5.5152    1.0000    
SURF96 R U T -1 55.000    3.7662    1.0000    
SURF96 R C T -1 55.000    5.3120    1.0000    
SURF96 R U T -1 50.000    3.7224    1.0000    
SURF96 R C T -1 50.000    5.1174    1.0000    
SURF96 R U T -1 48.000    3.6989    1.0000    
SURF96 R C T -1 48.000    5.0410    1.0000    
SURF96 R U T -1 46.000    3.6716    1.0000    
SURF96 R C T -1 46.000    4.9649    1.0000    
SURF96 R U T -1 44.000    3.6406    1.0000    
SURF96 R C T -1 44.000    4.8888    1.0000    
SURF96 R U T -1 42.000    3.6058    1.0000    
SURF96 R C T -1 42.000    4.8124    1.0000    
SURF96 R U T -1 40.000    3.5678    1.0000    
SURF96 R C T -1 40.000    4.7353    1.0000    
SURF96 R U T -1 38.000    3.5274    1.0000    
SURF96 R C T -1 38.000    4.6573    1.0000    
SURF96 R U T -1 36.000    3.4859    1.0000    
SURF96 R C T -1 36.000    4.5782    1.0000    
SURF96 R U T -1 34.000    3.4461    1.0000    
SURF96 R C T -1 34.000    4.4980    1.0000    
SURF96 R U T -1 32.000    3.4120    1.0000    
SURF96 R C T -1 32.000    4.4169    1.0000    
SURF96 R U T -1 30.000    3.3911    1.0000    
SURF96 R C T -1 30.000    4.3358    1.0000    
SURF96 R U T -1 29.000    3.3892    1.0000    
SURF96 R C T -1 29.000    4.2958    1.0000    
SURF96 R U T -1 28.000    3.3961    1.0000    
SURF96 R C T -1 28.000    4.2567    1.0000    
SURF96 R U T -1 27.000    3.4151    1.0000    
SURF96 R C T -1 27.000    4.2190    1.0000    
